mrp_production_real_cost/models/mrp_production_workcenter_line.py

```diff
@@ -97,6 +98,8 @@ def action_pause(self):
 
     @api.multi
     def action_done(self):
+        not_paused_records = self.filtered(lambda x: x.state != 'pause')
+        not_paused_records._write_end_date_operation_line()
```

Comment:
When a workorder is paused the end_date is written, if I do not do this, it will be changed.